scripts/lib/Utils.psm1

```diff
@@ -282,6 +282,20 @@ Function registerTest($testname, $index, $bucket, $filter, $moreParams, $cluster
     comm
 }
 
+Function Kill-Children ($Pid, $SessionId)
+{
+    Get-WmiObject win32_process | Where {$_.ParentProcessId -eq $Pid -And $_.SessionId -eq $SessionId -And -Not [string]::IsNullOrEmpty($_.Path) }) | ForEach-Object { Kill-Children $_.ProcessId $_.SessionId }
+    If (Get-Process -Id $Pid -ErrorAction SilentlyContinue)
+    {
+        Write-Host "Killing child: $Pid"
+
+        # Try to avoid https://wiki.jenkins.io/display/JENKINS/Spawning+processes+from+build:
+        handle.exe -p $Pid | Where {$_ -match "^\s*([0-9A-F]+): File..*$" } | ForEach { $h = $Matches[1]; handle.exe -c $h -p $Pid -y | Out-Null}
```

Comment:
there really is no powershell internal version of `handle.exe`?

Reply:
There is something custom: https://www.powershellgallery.com/packages/PoshInternals/1.0.34/Content/Handle.ps1